recce/event/track.py

```diff
@@ -109,6 +114,12 @@ def invoke(self, ctx: Context) -> t.Any:
             if branch is not None:
                 props['branch'] = sha256(branch.encode()).hexdigest()
 
+            if target_path is not None:
```

Comment:
Hi even,

The `target_path` and `target_base_path` here contain default values `target` and `target-base`.
So you would still need to compare with the hashed `target` and hashed `target-base` value to check if the customed path was given.

Is it what you expect? If yes, then it's all good to me.

Reply:
Yes, I want to know if it's different and how many different kind of different name people use, but I don't want to collect sensitive data.

So I hash it and hash the default value while analyzing it.